### ISSUE TITLE
Plugin fails if /calico key is missing from etcd.

### ISSUE THREAD

**Issue by Lukasa** [Metaswitch/calico#308]
If a totally clean etcd installation is used, no Calico component will start up, because they all complain that the `/calico` key is not present. At least one component needs to be capable of taking action to insert that key!

**Comment by Lukasa** [comment]
Here's the traceback:

```
2015-04-02 14:50:06.127 18239 ERROR calico.openstack.mech_calico [-] Exception in periodic resync thread
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico Traceback (most recent call last):
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico   File "/usr/local/lib/python2.7/dist-packages/calico-0.14-py2.7.egg/calico/openstack/t_etcd.py", line 73, in periodic_resync_thread
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico     self.resync_endpoints()
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico   File "/usr/local/lib/python2.7/dist-packages/calico-0.14-py2.7.egg/calico/openstack/t_etcd.py", line 96, in resync_endpoints
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico     r = self.client.read('/calico/host', recursive=True)
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico   File "/usr/local/lib/python2.7/dist-packages/python_etcd-0.3.2-py2.7.egg/etcd/client.py", line 344, in read
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico     self.key_endpoint + key, self._MGET, params=params, timeout=timeout)
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico   File "/usr/local/lib/python2.7/dist-packages/python_etcd-0.3.2-py2.7.egg/etcd/client.py", line 582, in api_execute
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico     return self._handle_server_response(response)
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico   File "/usr/local/lib/python2.7/dist-packages/python_etcd-0.3.2-py2.7.egg/etcd/client.py", line 598, in _handle_server_response
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico     etcd.EtcdError.handle(**r)
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico   File "/usr/local/lib/python2.7/dist-packages/python_etcd-0.3.2-py2.7.egg/etcd/__init__.py", line 142, in handle
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico     raise exc(msg)
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico KeyError: 'Key not found : /calico'
2015-04-02 14:50:06.127 18239 TRACE calico.openstack.mech_calico

```

**Comment by Lukasa** [comment]
Fixed in #310.
